### DISCUSSION TITLE
Continue to be facing 'initialize failure' on Codespace

### DISCUSSION THREAD

**Discussion by ThinkinGim** [community/community#24977] [Codespaces]
<p>I’m facing ‘Initializing environment failed’ for all personal codespace of mine. There is no info to address on this. Could you anyone let me know how to resolve this kind of situation?</p>

**Comment by matthewisabel** [comment]
<p>Hi, sorry you’re hitting this and thanks for reporting this. We’re seeing a few cases of this error we’re working to track down at the moment. We’ll follow up when we have an update or if we need to get additional information.</p>

**Comment by jkeech** [comment, marked as answer]
<p><a class="mention" href="/thinkingim">@ThinkinGim</a>, can you retry and let us know if you still face the same issue?</p>

**Comment by ThinkinGim** [comment]
<p>It’s came back to normal. Thank you for the inform.</p>

**Comment by sainathkollipara** [comment]
<p>I started facing this issue from few hours, any quick resolution around this?</p>
<p><img src="https://user-images.githubusercontent.com/5463341/181087984-3f172940-7ba4-42d7-9b8d-210d1a580f15.png" alt="image" data-base62-sha1="shbi0ehO5wYiCvbP1desYbmoVEQ" width="512" height="194"></p>

**Comment by fugufish** [comment]
<p>same here, no matter how many times I retry</p>

**Comment by scatyf3** [comment]
My friend facing same issue in 2023 :(. My friend tried delect the codespace and restart new one, however, every codespace on her account initialize failed. Is there any approach to fix this bug?

**Reply by ayinde-xyz** [reply]
I'm also facing the same issue right now
